### PR TITLE
clean up UI fields to match synopsysctl

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1074#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@mphammer: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1074#issuecomment-553465310):

>/cherry-pick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: pkg/polaris/ctl_polaris.go [hunk 1]
```diff
@@ -122,16 +122,16 @@ func (ctl *CRSpecBuilderFromCobraFlags) SetPredefinedCRSpec(specType string) err
 func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Command, master bool) {
 
 	// [DEV NOTE:] please organize flags in order of importance
-	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris\n")
+	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris")
```
Comment (yashbhutwala) [inline pkg/polaris/ctl_polaris.go]: can we keep the new lines?  These are meant to act as grouping for now 😄

Comment (mphammer) [reply pkg/polaris/ctl_polaris.go]: Done!

**Comment by operator-r2d2** [comment]
@mphammer: #1074 failed to apply on top of branch "master":
```Applying: clean up UI fields to match synopsysctl
error: Failed to merge in the changes.
Using index info to reconstruct a base tree...
M	pkg/synopsysctl/a_synopsysctl-packr.go
M	pkg/synopsysctl/cmd_serve_UI.go
Falling back to patching base and 3-way merge...
Auto-merging pkg/synopsysctl/cmd_serve_UI.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_serve_UI.go
Auto-merging pkg/synopsysctl/a_synopsysctl-packr.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/a_synopsysctl-packr.go
Patch failed at 0001 clean up UI fields to match synopsysctl

```

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1074#issuecomment-553465310):

>/cherry-pick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@mphammer: cannot checkout masterr: error checking out masterr: exit status 1. output: error: pathspec 'masterr' did not match any file(s) known to git.


<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1074#issuecomment-554039706):

>/cherry-pick masterr


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@mphammer: new pull request created: #1078

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1074#issuecomment-554039863):

>/cherry-pick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**